src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1685,12 +1705,19 @@ namespace UnitarySystems {
                 sizerCoolingCapacity.initializeWithinEP(state, CompType, CompName, PrintFlag, RoutineName);
                 CoolCapAtPeak = sizerCoolingCapacity.size(state, TempSize, errorsFound);
                 state.dataSize->DXCoolCap = CoolCapAtPeak;
-                EqSizing.CoolingCapacity = true;
-                EqSizing.DesCoolingLoad = CoolCapAtPeak;
+                // CoilSystem does not size the cooling coil (#8761)
+                if (this->UnitType == "AirLoopHVAC:UnitarySystem") {
+                    EqSizing.CoolingCapacity = true;
+                    EqSizing.DesCoolingLoad = CoolCapAtPeak;
+                }
```

Comment:
Another difference based on parent type. These must get resolved so that these types of checks don't get spread throughout UnitarySystem. Also need an enum here, I got lazy when trying to fix diffs.